openedx/features/enterprise_support/utils.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import unicode_literals
+
+import hashlib
+
+import six
+
+
+def get_cache_key(**kwargs):
+    """
+    Get MD5 encoded cache key for given arguments.
+
+    Here is the format of key before MD5 encryption.
+        key1:value1__key2:value2 ...
+
+    Example:
+        >>> get_cache_key(site_domain="example.com", resource="enterprise-learner")
+        # Here is key format for above call
+        # "site_domain:example.com__resource:catalogs"
```

Comment:
`"site_domain:example.com__resource:enterprise-learner"` to match the call above.